Makefile

```diff
@@ -35,8 +43,16 @@ helm-install:
 helm-template:
 	helm template kubernetes-sidecar-injector ./charts/kubernetes-sidecar-injector
 
+kind-create:
+	-kind create cluster --image "${KIND_REPO}:${KUBE_VERSION}" --name ${KIND_CLUSTER}
+
 kind-install: kind-load helm-install
 
+kind: kind-create kind-install
+
+follow-logs:
+	kubectl logs -n sidecar-injector deployment/kubernetes-sidecar-injector --follow
+
 install-sample-container:
```

Comment:
Not specifically in this commit but 
```
          "docker.io/expediagroup/haystack-agent:latest": failed to resolve reference
          "docker.io/expediagroup/haystack-agent:latest": pull access denied, repository
          does not exist or may require authorization: server message: insufficient_scope:
```

```
docker pull docker.io/expediagroup/haystack-agent:latest
Error response from daemon: pull access denied for expediagroup/haystack-agent, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```


```
sample               inject-container-8d559dd7f-8lfzn                 1/2     ImagePullBackOff   0          109s
sample               inject-init-container-d7dc86c8f-g6cm9            1/1     Running            0          103s
```

Reply:
This was pre-existing i guess.

Reply:
this is from the legacy repo example, i haven't had a chance to change it to something better.